syncer/syncer.go

```diff
@@ -51,6 +54,7 @@ func NewSyncer(
 		syncPeerService: NewSyncPeerService(network, blockchain),
 		syncPeerClient:  NewSyncPeerClient(logger, network, blockchain),
 		blockTimeout:    blockTimeout,
+		newStatus:       make(chan struct{}),
```

Comment:
Please add `Ch` in suffix of the name

Reply:
Agree, fixed 3866ac73c466ae445be4ade0cfedb447a3d52066